pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -108,6 +109,32 @@ func (k *KubeASCheck) Run() error {
 	if err != nil {
 		k.warn("could not collect API Server component status: ", err.Error())
 	}
+
```

Comment:
Do you think the event collection logic should be behind a `collect_events` option?

Reply:
yeah eventually I want to add the option along with the filters on kind/type etc